site/content/en/docs/Getting started/installation.md

```diff
@@ -50,14 +50,14 @@ Commands
                 Workflows are potentially-dynamic graphs of computational tasks to execute.
 
   Settings ⚙️
-    version     Print the version number.
     configure   Commands for configuration.
                 Configuration is stored per user.
 
 Flags
-  -h, --help      help for agc
-  -v, --verbose   Display verbose diagnostic information.
-      --version   version for agc
+      --format string   Format option for output. Valid options are: text, tabular (default "text")
```

Comment:
This is not entirely correct. the right supported types are text and *table*:

```
$ agc_110 context list --format table
2021-11-11T12:16:51-08:00 𝒊  Listing contexts.
Name		EngineName
miniContext	miniwdl
myContext	cromwell
spotCtx		cromwell

```

Reply:
I'll update our help text in the CLI code to fix this